### DISCUSSION TITLE
`Play music` shortcut key shouldn't available in editor

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#19398] [Q&A]
I add `/` for `Play music` shortcut(more like stable)

The default key `F3` is already covered by `Timing section`

![image](https://user-images.githubusercontent.com/34775378/180949977-cf1c742e-49ee-49ac-aa8d-1b4c16457e0b.png)

https://user-images.githubusercontent.com/34775378/180949988-cbab71d1-ff9c-469b-af6d-22e64489b4a5.mp4

2022.723.0

Even without shortcut keys, just use `now playing` panel also cause this problem

**Comment by frenzibyte** [comment]
I'll note that F3 for playing/pausing music will likely not stay, see https://github.com/ppy/osu/issues/12776 for more information.

**Comment by peppy** [comment]
Close the now playing overlay.

**Reply by cdwcgt** [reply]
still exists when closed

**Reply by peppy** [reply]
Thanks, I better understand the issue now.

Gameplay gets around this only by luck (the fact it tweens the tracks' frequency to zero means that the music control overlay can't unpause it). I'll keep tracking this one as I work through #11400.
